src/client/runner/automation/utils/get-element.js

```diff
@@ -0,0 +1,73 @@
+import hammerhead from '../../deps/hammerhead';
```

Comment:
I don't think it should be in `utils` since it's not some generic thing, but some piece of business logic that is tightly coupled with the automations. So just put in `src/client/runner/automation`